sample/Getting started.jl

```diff
@@ -13,23 +13,42 @@ macro bind(def, element)
     end
 end
 
+# ╔═╡ 9b62f2c6-8037-486f-b8ec-a95b577b8d74
+# Code targeted for potential inclusion in the Pluto.jl package:
+# Macro could also be named md2, or something else...
+begin
+	import Pkg; Pkg.add("CommonMark")
+	using CommonMark
+	
+	parser = Parser()
+	enable!(parser, MathRule())
+	enable!(parser, TableRule())
+	
+	function cm_md(str_ex)
+		ast = parser(str_ex)
+		html_ex = CommonMark.html(ast)
+		HTML(html_ex)
+	end
```

Comment:
You _should_ be able to just do something like

```julia
begin
    using CommonMark

    macro cm_str(str)
        return Parser()(str)
    end
end
```

since the AST object that `CommonMark` returns implements `show` for `MIME"text/html`, unless there's something Pluto-specific that requires something else?

Reply:
Thanks! Between this tip and looking at the existing markdown lib code, I was able to cobble something together that works with math syntax (e.g. properly escaped LaTeX) as well. :)